pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -131,6 +131,13 @@ class Meta:
         )
 
 
+class CollectionNamespaceSerializer(serializers.Serializer):
+    """
+    A serializer for a Collection Version namespace field.
+    """
+    name = serializers.CharField(max_length=32, source="namespace")
```

Comment:
For serializer that doesn't actually validate data `max_length` is not necessary.

Reply:
updated